controller/Lists.php

```diff
@@ -531,14 +539,24 @@ private function getListData(bool $showRemoteLists = false): array
                 'uri' => tao_helpers_Uri::encode($listClass->getUri()),
                 'label' => $listClass->getLabel(),
                 'editable' => $listService->isEditable($listClass),
-                'elements' => $listElements->jsonSerialize(),
+                'elements' => $this->getSortedElementsDependingOnListClass($listClass, $listElements),
```

Comment:
please do not forget to update tao-core on composer.json file.

Reply:
I have this item in checklist in PR description in order to not forget to do it when I'd be ready to merge.

Reply:
https://github.com/oat-sa/extension-tao-backoffice/pull/127/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R26